docs/expo-dict.json

```
@@ -11,6 +11,7 @@
   "expo sdk": "Expo SDK",
   "ios simulator": "iOS Simulator",
   "mac os": "macOS",
+  "objective c": "Objective-C",
```

We can also add `objc` and `obj-c` 😉
```suggestion
  "objective c": "Objective-C",
  "objc": "Objective-C",
  "obj-c": "Objective-C",
```

---

Those short forms might produce the false-positives due to code blocks highlight language keyword (`objc`), will need to test this locally first.

---

Those rules did not clash with the code block, but also do not work correctly, since they do not report `objc` or `obj-c` usages anyway. Will need to research this behaviour a bit more, but for now I'm going to skip those entries.